src/dune_rpc_impl/client.mli

```diff
@@ -0,0 +1,6 @@
+open Import
+
+include
+  Dune_rpc.Client.S
+    with type 'a fiber := 'a Fiber.t
+     and type chan := Csexp_rpc.Session.t
```

Comment:
An mli for this has been added to https://github.com/ocaml/dune/pull/5888 as well.